README.md

```diff
@@ -18,7 +18,7 @@ The original idea for WTFJS belongs to [Brian Leroux](https://twitter.com/brianl
 You can install this handbook using `npm`. Just run:
 
 ```
-$ npm install -g wtfjs
```

Comment:
Please, do not change anything in the main README, except for adding a link to your translation. Let's not fight over formatting preferences.

Reply:
That my mistake. so I should change back on my repository (**NantipatSoftEn/wtfjs**) right? 

This is my first pull request to contribute, Can you suggest to me plz what to do?

Also,  about find someone to check your translation. that means need to invite someone to this repository right?

Thank you